test/cets_SUITE.erl

```diff
@@ -1054,19 +1056,46 @@ status_unavailable_nodes(Config) ->
     ok = cets_discovery:wait_for_ready(DiscoName, 5000),
     ?assertMatch(#{unavailable_nodes := ['badnode@localhost']}, cets_status:status(DiscoName)).
 
+status_unavailable_nodes_is_subset_of_discovery_nodes(Config) ->
+    Node1 = node(),
+    Me = self(),
+    %% We use our process dictionary to set disco nodes dynamically
+    put(disco_nodes, [Node1, 'badnode@localhost']),
+    F = fun(State) ->
+        {dictionary, Dict} = erlang:process_info(Me, dictionary),
+        Nodes = proplists:get_value(disco_nodes, Dict),
+        {{ok, Nodes}, State}
+    end,
```

Comment:
For me using `meck` would be more intuitive and easier to understand that we are mocking the discovery backend.
```suggestion
    meck:new(disco_backend, [non_strict]),
    meck:expect(disco_backend, get_disco_nodes, [Node1, 'badnode@localhost']),
```

Reply:
Sure, but because we run tests in parallel, module name would have to be generated from the testcase name.
We would have to use `make_name(Config)`, instead of disco_backend.

Because if we copy the testcase (like to test a similar situation), it would stop working.